packages/ui/components/Calendar.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import * as React from "react";
+import { ChevronLeft, ChevronRight } from "lucide-react";
+import { DayPicker } from "react-day-picker";
```

Comment:
Why did you decide to use React Day Picker instead of shadcn day picker?

Reply:
shadcn date picker is using react-day-picker internally.